tests/zeromq.terminal/Program.cs

```diff
@@ -14,67 +13,68 @@ namespace zeromq.terminal
     ///</summary>
     class Program
     {
-        static void Main(string[] args)
+        static async Task Main(string[] args)
         {
             var version = args.FirstOrDefault();
             var configuration = SocketConfiguration.InprocConfig("this-inproc-sir-" + Guid.NewGuid());
             
             if (version != null)
```

Comment:
dammit ... I'm totally a sucker for C# 9.0 `is not null` 😉

Reply:
I dont get it? Did you intend to comment on line 21?
So for me this is the usual style debate, so I go with yours instead and will update :D

Reply:
yes, I did ... but as we discussed here https://github.com/Flash0ver/F0.Analyzers/issues/63#issuecomment-813416206 it's indeed a matter of style ... I justed wanted to voice my personal preference, but I'd be super fine to use "classic null checks" as well ... I can get used to it to keep styling consistent 😉